usd/src/usd_parser/USDMaterial.cc

```diff
@@ -44,6 +44,13 @@ namespace sdf
     UsdErrors errors;
     if (ignition::common::exists(_ori))
     {
+      // If the file exists then we should remove it and we will copy the
+      // file below
+      if (ignition::common::exists(_dest))
+      {
+        ignition::common::removeFile(_dest);
+      }
```

Comment:
Is there a chance that this code removes a file that is still needed at some point later on?

Reply:
When you try to use `ignition::common::copyFile` and the file exists this will return false and it generates an error. I prefer to remove the existing file and void the error.

This may happen when you try to run several times the same command with the same file in the same folder.

Reply:
done here https://github.com/ignitionrobotics/sdformat/pull/895/commits/d529bf4b3692569cd9b5db801e1ce13b65c6cc09
I don't know if there is an easy way to do it

Reply:
I think the approach you took in https://github.com/ignitionrobotics/sdformat/commit/d529bf4b3692569cd9b5db801e1ce13b65c6cc09 is pretty good. I left a few review comments about it below.